node_modules/oae-principals/lib/rest.user.js

```diff
@@ -222,7 +222,8 @@ OAE.globalAdminRouter.on('post', '/api/user/:tenantAlias/create', function(req,
         'locale': req.body.locale || req.ctx.locale(),
         'timezone': req.body.timezone,
         'publicAlias': req.body.publicAlias,
-        'acceptedTC': (req.body.acceptedTC === 'true')
+        'acceptedTC': (req.body.acceptedTC === 'true'),
+        'emailVerified': false
```

Comment:
Does that mean that users created by the global admin will immediately receive an email asking them to confirm their email address?